lug/config.yaml

```diff
@@ -234,3 +234,17 @@ repos:
     interval: 4800
     name: rust-static
     <<: *oneshot_common
+  # pypi
+  - type: shell_script
+    script: /app/mirror-clone pypi --target http://siyuan-mirror-intel:8000/pypi-packages
```

Comment:
Why is pypi added here? If it's intended could you update the description of this PR?

Reply:
Thanks for reminding. I've updated the PR.